lib/fetch/webidl.js

```diff
@@ -399,6 +399,11 @@ webidl.nullableConverter = function (converter) {
 
 // https://webidl.spec.whatwg.org/#es-DOMString
 webidl.converters.DOMString = function (V, opts = {}) {
+  // Note: avoid re-stringify
+  if (typeof V === 'string') {
+    return V
+  }
```

Comment:
The String constructor calls [`ToString(value)`](https://tc39.es/ecma262/#sec-tostring) in which the first step is `If argument is a String, return argument.` This almost certainly won't improve performance in a meaningful way (and if it does, there's likely somewhere else it could be improved, like initializing opts to undefined).

Reply:
Hello!
I looked into it and Deno already does that.
Please let me know again if I should change it.
https://github.com/denoland/deno/blob/621440017a3fb93dcba6f9153ad150994592de02/ext/webidl/00_webidl.js#L386C66-L386C66

Reply:
I'd prefer if it was removed